src/main/java/com/powsybl/openloadflow/network/impl/PropagatedContingency.java

```diff
@@ -264,8 +264,15 @@ private static boolean isCoupler(Switch s) {
         return c1 != c2 && c1.getType() == IdentifiableType.BUSBAR_SECTION && c2.getType() == IdentifiableType.BUSBAR_SECTION;
     }
 
-    public Optional<LfContingency> toLfContingency(LfNetwork network, GraphDecrementalConnectivity<LfBus> connectivity,
-                                                   boolean useSmallComponents) {
+    public Optional<LfContingency> toLfContingencyForSensitivityAnalysis(LfNetwork network) {
```

Comment:
@geofjamg and @flo-dup it seems that @geofjamg have met some issues with the `useSmallComponents` boolean. Nevertheless, we have a lot of PR after this one, because @phiedw is making some real tests on AC sensitivity analysis. So maybe we can merge this PR and write an issue about the `useSmallComponents` issue?